### ISSUE TITLE
Unable to get meta data for data sets shared with me

### ISSUE THREAD

**Issue by smach** [domoinc/rdomo#5]
Hi. I'm just wondering if it will be possible to get meta data with ds_meta() for data sets that have been shared with me? Right now I can only get info for data sets that I own but not those that have been shared with me.

**Comment by jeremydmorris** [comment]
Do you happen to know what your user role is in Domo?

**Comment by smach** [comment]
@jeremydmorris I think it's below admin but above basic user. Is there some way to see that in my profile? I looked but can't find it. Thanks!

**Comment by jeremydmorris** [comment]
I might try using the users_list function and looking yourself up there. That user list tells you the role for each user. It would also be interesting to see if your permissions allow you to get that data. Ultimately, the data returned by the API depends on your permission level in Domo. 

One other question, can you use ds_get on data sets that are shared with you and it's just the meta data you can't get? 

Either way, we may need to have a discussion with Domo engineering since this is more an API issue than an issue with this package.

**Comment by smach** [comment]
Yes I can use `ds_get() `on shared data sets I don't own. I can also run `users_list()` (I'm "Privileged"). Thanks.
